contracts/managers/TwoAssetStrategyManager.sol

```diff
@@ -263,4 +347,31 @@ contract TwoAssetStrategyManager {
         return block.timestamp >= lastRebalanceTimestamp.add(rebalanceInterval) &&
             rebalancingSetTokenInstance.rebalanceState() == RebalancingLibrary.State.Default;        
     }
+
+    /*
+     * Return if enough time passed since last initialTrigger
+     *
+     * @return       Whether enough time has passed since last initialTrigger
```

Comment:
@return and the comment above is repetitive. Would be cool with removing the @return